Assets/Scripts/UI/CameraIntMsg.cs

```diff
@@ -3,41 +3,39 @@
 
 public class CameraIntMsg : MonoBehaviour {
 
-    public GameObject Camera;
+    public GameObject CameraSprite;
     public GameObject ArrowsSide;
     public GameObject ArrowsUp;
-    public Controller2 isghostMode;
+    private Controller2 ghostMode;
     private bool onlyOnce;
 
     // Use this for initialization
     void Start()
     {
         onlyOnce = false;
+        ghostMode = GetComponentInChildren<Controller2>();
```

Comment:
@Gowtham100 I'm assuming that this script is attached to the camera, and whenever some key is pressed, you can see the UI? Next time do explain what the scripts are for, cos im making a lot of assumptions here